src/libraries/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -185,6 +185,16 @@ public void GetFile_InMemory()
             Assert.Throws<FileNotFoundException>(() => asm.GetFiles(getResourceModules: false));
         }
 
+        [Fact]
+        public void CodeBase_InMemory()
+        {
+            var inMemBlob = File.ReadAllBytes(SourceTestAssemblyPath);
+            var asm = Assembly.Load(inMemBlob);
+#pragma warning disable SYSLIB0012
+            Assert.Throws<PlatformNotSupportedException>(() => asm.CodeBase);
```

Comment:
For non-single-file this should not throw - right? I don't understand this test...

Reply:
Test is wrong now. I originally was throwing from everyhting that was in-memory before I figured out we actually doc'd the behavior. I need to write a new test using single-file, but don't know where to write the test

Reply:
Based on [this comment](https://github.com/dotnet/runtime/pull/40974#issuecomment-675618979) I think this is a placeholder, and the single-file test is TBD.  

But I think `CodeBase_InMemory` should check `Assert.NotEmpty(asm.CodeBase)`; and the single-file test should be added to [AppHost.Bundle](https://github.com/dotnet/runtime/tree/master/src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests) tests.

Reply:
> I need to write a new test using single-file, but don't know where to write the test

@agocke I think the easiest way to add a single-file test for this is to follow the BundleProbe test in AppHost.Bundle.Tests.